### DISCUSSION TITLE
Can we set Marker Cluster options?

### DISCUSSION THREAD

**Discussion by fprl** [inocan-group/vue3-google-map#226] [Q&A]
Hi there,

Is it possible to pass an object with this options? I don't see it in the component API.

<img width="770" alt="image" src="https://user-images.githubusercontent.com/68303277/202679964-ee63ec80-b39e-4351-89a1-cb09e939be4d.png">

Regards,

**Comment by HusamElbashir** [comment]
Hi @francoromanol 

Unfortunately no. We're using a different library for clustering:
https://www.npmjs.com/package/@googlemaps/markerclusterer

**Comment by fprl** [comment]
Hi @HusamIbrahim , thanks for the response. I found a way to do it with ref and computed.

One last question, is it possible to use InfoWindow inside CustomMarker? Its not working for me.

Regards

**Comment by HusamElbashir** [comment]
No this isn’t supported unfortunately.
